subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -1090,7 +1090,7 @@ static void ticker_cb(u32_t ticks_at_expire, u32_t remainder, u16_t lazy,
 		u32_t random_delay;
 		u32_t ret;
 
-		lll_entropy_get(sizeof(random_delay), &random_delay);
+		lll_rng_isr_get(&random_delay, sizeof(random_delay));
```

Comment:
This is bug fix, LHS should have been lll_entropy_isr_get (in the first place).

Reply:
It's a name only change, `lll_entropy_isr_get` did not exist, but `lll_entropy_get ` called the entropy_isr version.